Tungsten compilation error (Texture.java is missing Texture.Format.RGBM enumeration and Texture.Builder.rgbm method)

**Describe the bug**
When trying to compile Tungsten (e.g. `./gradlew :standalone:run`), there is following error:
```
e: /home/dev/filament-1.3.1/tools/tungsten/core/src/com/google/android/filament/tungsten/ui/preview/Ibl.kt: (124, 70): Unresolved reference: RGBM
e: /home/dev/filament-1.3.1/tools/tungsten/core/src/com/google/android/filament/tungsten/ui/preview/Ibl.kt: (132, 18): Unresolved reference: rgbm

FAILURE: Build failed with an exception.
```


**To Reproduce**
Follow the instruction reported in the Tungsten README.md documentation (also available at https://github.com/google/filament/tree/master/tools/tungsten)

Steps to reproduce the behavior:
1. get sources either from 1.3.1 release or from git
1. compile and install (e.g. `ninja install`) and verify that the `dist` directory contains the files required by the "prerequisites" section in the README.md
2. `cd tools/tungsten`
3. either execute `./gradlew :standalone:run` or `./gradlew :core:build`


**Expected behavior**
Compilation of the tungsten tools should complete.
Indeed, looking at the `Texture.java` source code, both the `Texture.Format.RGBM` enumeration and `Texture.Builder.rgbm()` method are missing.

**Desktop**
Tested on:
- Windows 10 (clang 7.0.1, ninja 1.9.0, cmake 3.15.0, java 1.8.0_151, gradle 4.6) 
- Ubuntu 18.04.3 (clang 7.0.0-3~ubuntu0.18.04.1, ninja 1.8.2, cmake 3.10.2, java 1.8.0_222, , gradle 4.6)

--- a/tools/tungsten/core/src/com/google/android/filament/tungsten/ui/preview/Ibl.kt
+++ b/tools/tungsten/core/src/com/google/android/filament/tungsten/ui/preview/Ibl.kt
@@ -87,7 +87,7 @@ internal class Ibl(val engine: Engine, private val pathPrefix: String) {
         val faceOffsets = IntArray(6)
 
         val rawBuffers = cubemapFaces.map { face ->
-            val path = "$pathPrefix/$facePrefix$face.rgbm"
+            val path = "$pathPrefix/$facePrefix$face.rgb32f"
             val stream: InputStream = javaClass.classLoader.getResourceAsStream(path)
                     ?: throw RuntimeException("Could not get stream for cubemap face $path.")
 
@@ -121,15 +121,14 @@ internal class Ibl(val engine: Engine, private val pathPrefix: String) {
         buffer.position(0)
 
         val bufferDescriptor =
-                Texture.PixelBufferDescriptor(buffer, Texture.Format.RGBM, Texture.Type.UBYTE)
+                Texture.PixelBufferDescriptor(buffer, Texture.Format.RGB, Texture.Type.UBYTE)
 
         // If the texture hasn't been created yet, create it.
         val resultTexture = texture ?: Texture.Builder()
                 .width(size)
                 .height(size)
                 .levels(levels)
-                .format(Texture.InternalFormat.RGBA8)
-                .rgbm()
+                .format(Texture.InternalFormat.R11F_G11F_B10F)
                 .sampler(Texture.Sampler.SAMPLER_CUBEMAP)
                 .build(engine)
 

